packages/connect/e2e/__fixtures__/getAddressMultisigPubkeysOrder.ts

```diff
@@ -0,0 +1,119 @@
+import { MultisigPubkeysOrder } from '@trezor/protobuf/src/messages';
+
+// https://github.com/trezor/trezor-firmware/blob/45a029edcc720b0825e4a13504d460faada8b72c/tests/device_tests/bitcoin/test_getaddress.py
+
+const nodeInternal = {
+    // trezorctl btc get-public-node -n m/45h/0
+    node: 'xpub69numS2foCPDWvbnTE95o7m92kWr1CFoSA91EHAADBDHcaQw6VwrDFbg5KUGGnyLdKjJ3ohRG4BHQNzpM1kBbVPKkBQB5j221NcvByaWDz2',
+    address_n: [0, 0],
+};
+
+const nodeExternal = {
+    // trezorctl btc get-public-node -n m/45h/1
+    node: 'xpub69numS2foCPDZmjusERkaVyVhRAQXzCjpYExzFSEHCV8615UzJDnCeHDU8av9eXoaWR5CESEymVz1KBhpgFCpLg22jA6msZ4VGWaUXcSSDy',
+    address_n: [0, 0],
+};
+const pubkeysOrder1 = [nodeInternal, nodeExternal];
+
+const pubkeysOrder2 = [nodeExternal, nodeInternal];
+
+const multisigUnsorted1 = {
+    pubkeys: pubkeysOrder1,
+    signatures: ['', ''],
+    m: 2,
+    pubkeys_order: MultisigPubkeysOrder.PRESERVED,
+};
+
+const multisigUnsorted2 = {
+    pubkeys: pubkeysOrder2,
+    signatures: ['', ''],
+    m: 2,
+    pubkeys_order: MultisigPubkeysOrder.PRESERVED,
```

Comment:
so I understand that `pubkeys_order` doesn't need to be added anywhere in the runtime, because it s part of `multisig` field which is already passed to the device.

Reply:
Right.